std_msgs/Header header
Pose2D[] points
int32[] labels